src/portable/synopsys/dwc2/dwc2_esp32.h

```diff
@@ -82,6 +82,22 @@ static const dwc2_controller_t _dwc2_controller[] =
 #endif
 };
 
+static ep_avaliable_count_t ep_avaliable_count[] =
```

Comment:
This structure will be: 
```suggestion
static dwc_ep_config_t dwc_ep_config[] = 
{
#ifdef DWC2_FS_PERIPH_BASE
  { .out_ep = 0, .in_ep = 0, .ep_max_count = DWC2_FS_EP_MAX },
#endif

#ifdef DWC2_HS_PERIPH_BASE
  { .out_ep = 0, .in_ep = 0, .ep_max_count = DWC2_HS_EP_MAX },
#endif
};
```